src/main/resources/io/jenkins/plugins/extendedsecuritysettings/ExtendedSecuritySettings/config.jelly

```diff
@@ -31,5 +31,12 @@
         <f:entry field="enableXssProtectionHeader">
             <f:checkbox title="${%enableXssProtectionHeader}"/>
         </f:entry>
+        <f:entry title="${%httpHeaderNames}" help="${descriptor.getHelpFile('httpHeaderNames')}">
+            <f:repeatableProperty field="httpHeaderNames" add="${%addFilter}">
```

Comment:
In the mentionned doc, it's written you need to use:
```
<f:entry title="">
   <div align="right">
      <f:repeatableDeleteButton />
   </div>
</f:entry>
```

Without the `f:entry`, your delete button will be place at the top of your field which is weird.

<details>

<summary>Screenshot of current version</summary>

![Screenshot_2019-09-04_085017_001](https://user-images.githubusercontent.com/2662497/64233228-ecfeb100-cef3-11e9-9d47-f642841ddaa3.png)

</details>

<details>

<summary>Screenshot of "corrected" version</summary>

![Screenshot_2019-09-04_085055_001](https://user-images.githubusercontent.com/2662497/64233238-f12ace80-cef3-11e9-86ff-867a38fe965b.png)

</details>